core/src/main/java/nl/nn/adapterframework/pipes/WsdlGeneratorPipe.java

```diff
@@ -37,13 +39,17 @@
  */
 public class WsdlGeneratorPipe extends FixedForwardPipe {
 	private String from = "parent";
+	private @Setter AdapterManager adapterManager;
```

Comment:
Waarom niet gewoon 'Configuration'?

Reply:
De configuratie vraagt het weer aan de AdapterManager, dit is direct? Verder maakt het niet veel uit.

Reply:
Een pipe zou niet moeten weten dat er zoiets als een adaptermanager bestaat, lijkt me